### ISSUE TITLE
FFD Thickness 2D Read as Camber

### ISSUE THREAD

**Issue by aerialhedgehog** [su2code/SU2#153]
The Config --

```
DV_KIND= FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_CAMBER_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D, FFD_THICKNESS_2D
DV_MARKER= ( airfoil )
DV_PARAM= ( AIRFOIL, 1.0) ; ( AIRFOIL, 2.0) ; ( AIRFOIL, 3.0) ; ( AIRFOIL, 4.0) ; ( AIRFOIL, 5.0) ; ( AIRFOIL, 6.0) ; ( AIRFOIL, 7.0) ; ( AIRFOIL, 8.0) ; ( AIRFOIL, 9.0) ; ( AIRFOIL, 10.0) ; ( AIRFOIL, 11.0) ; ( AIRFOIL, 12.0) ; ( AIRFOIL, 13.0) ; ( AIRFOIL, 14.0) ; ( AIRFOIL, 15.0) ; ( AIRFOIL, 16.0) ; ( AIRFOIL, 17.0) ; ( AIRFOIL, 18.0) ; ( AIRFOIL, 1.0) ; ( AIRFOIL, 2.0) ; ( AIRFOIL, 3.0) ; ( AIRFOIL, 4.0) ; ( AIRFOIL, 5.0) ; ( AIRFOIL, 6.0) ; ( AIRFOIL, 7.0) ; ( AIRFOIL, 8.0) ; ( AIRFOIL, 9.0) ; ( AIRFOIL, 10.0) ; ( AIRFOIL, 11.0) ; ( AIRFOIL, 12.0) ; ( AIRFOIL, 13.0) ; ( AIRFOIL, 14.0) ; ( AIRFOIL, 15.0) ; ( AIRFOIL, 16.0) ; ( AIRFOIL, 17.0) ; ( AIRFOIL, 18.0) 
DV_VALUE= 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1
```

From the Log -- 

```
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 1 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 2 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 3 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 4 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 5 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 6 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 7 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 8 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 9 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 10 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 11 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 12 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 13 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 14 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 15 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 16 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 17 )
FFD 2D (camber) <-> airfoil <-> 0 <-> ( AIRFOIL, 18 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 1 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 2 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 3 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 4 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 5 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 6 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 7 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 8 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 9 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 10 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 11 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 12 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 13 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 14 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 15 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 16 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 17 )
FFD 2D (camber) <-> airfoil <-> 0.1 <-> ( AIRFOIL, 18 )

```

**Comment by btracey** [comment]
Fixed by #154

**Comment by aerialhedgehog** [comment]
thanks brendan!
